README.md

```diff
@@ -297,7 +299,7 @@ For a longer overview of Julia's dependencies, see these [slides](https://github
 [gcc]:          http://gcc.gnu.org
 [clang]:        http://clang.llvm.org
 [gfortran]:     https://gcc.gnu.org/fortran/
-[curl]:         http://curl.haxx.se
+[cURL]:         http://curl.haxx.se
```

Comment:
link should probably be in lower case all 3 places otherwise the "automatically download external libraries" might not link correctly